src/PluginManager.js

```diff
@@ -1,8 +1,33 @@
+function isString( value ) {
+	return typeof value === 'string';
+}
+
+
 class PluginManager {
 	constructor() {
 		this.plugins = [];
 	}
 
+	/**
+	 * Load given plugins.
+	 *
+	 * @param {string[]} plugins Names of plugins to load.
+	 * @returns {Promise}
+	 */
+	load( plugins ) {
+		if ( !Array.isArray( plugins ) || !plugins.every( isString ) ) {
+			throw new TypeError( 'Parameter must be an array of strings' );
+		}
+
+		const loaded = plugins.map( ( plugin ) => {
+			const module = require( plugin );
```

Comment:
Is this necessary to assign require function to variable?

Reply:
True, it could be safely inlined.